src/tests/simple_tests/testdata/log_precompile/entry.asm

```diff
@@ -0,0 +1,37 @@
+        .text
+        .file	"Test_26"
+        .rodata.cst32
+        .p2align	5
+        .text
+        .globl	__entry
+    __entry:
+    .main:
+        add 10000, r0, r1
+        add 1000, r0, r10
+        ; put 2 in r2 (sha)
+        add 2, r0, r2
+        ; precompile call - address 0x1, AuxData (additional costs - 0 for now)
+        log.precompile r2, r3, r4
```

Comment:
I am not sure it has intended behavior

`log.precompile` actives `log::precompilecall` opcode, which accepts:

- src0 encodes precompile call abi: https://github.com/matter-labs/era-zkevm_opcode_defs/blob/109d9f734804a8b9dc0531c0b576e2a0f55a40de/src/definitions/abi/precompile_call.rs#L6
- src1 encodes precompile aux data (basically cost for executing it): https://github.com/matter-labs/era-zkevm_opcode_defs/blob/109d9f734804a8b9dc0531c0b576e2a0f55a40de/src/definitions/abi/precompile_call.rs#L52

the type of precompile (sha/keccak/etc) is derived from the `this` address when the precompile is executed. 


In other words here:
- src0 r2 has a rather strange value, but since the entry point does not have a valid precompile address it is ignored
- src1 is 0, i.e. no ergs cost is beared

Reply:
thanks, I didn't notice that this is an ABI there.
So how do you pass the information about the type of precompile that you want to call?

Reply:
ok, go it. fixed.